### PR TITLE
[008] Packaging

### PR DESCRIPTION
**Pull request by zicongma**

### CODE REVIEW

File: hypothesisfuzzer/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from .fuzzer import Fuzzer # noqa
+name = "hypothesisfuzzer"
```
Comment (jackel119) [inline hypothesisfuzzer/__init__.py]: Why is this here? Is this standard practice?

Comment (romdespoel) [reply hypothesisfuzzer/__init__.py]: Seems to be standard practice...
